nesta/packages/quality_check/quality_function.py

```diff
@@ -0,0 +1,83 @@
+from quality_calculations import *

```

Comment:
Very minor point, but there is a convention for import orders which is typically
```
import <built in python modules>

import <external packages>

import <internal code>
```